src/lib/paginator/paginator.html

```diff
@@ -0,0 +1,39 @@
+<div class="mat-paginator-page-length-select">
+  <div class="mat-paginator-items-per-page-label">
+    {{_intl.itemsPerPageLabel}}
+  </div>
+
+  <md-select *ngIf="_displayedPageLengthOptions.length > 1"
+             [ngModel]="pageLength"
+             [aria-label]="_intl.itemsPerPageLabel"
+             (change)="_changePageLength($event.value)">
+    <md-option *ngFor="let pageLengthOption of _displayedPageLengthOptions" [value]="pageLengthOption">
+      {{pageLengthOption}}
+    </md-option>
+  </md-select>
+
+  <div *ngIf="_displayedPageLengthOptions.length <= 1">{{pageLength}}</div>
+</div>
+
+<div class="mat-paginator-range-label">
+  {{_intl.getRangeLabel(currentPageIndex, pageLength, listLength)}}
+</div>
+
+<button md-icon-button
+        class="mat-paginator-navigation-button mat-paginator-navigation-previous"
+        (click)="navigateToPreviousPage()"
+        [attr.aria-label]="_intl.previousPageLabel"
+        [mdTooltip]="_intl.previousPageLabel"
+        [mdTooltipPosition]="'above'"
+        [disabled]="!canNavigateToPreviousPage(-1)">
+  <div class="mat-paginator-increment"></div>
+</button>
+<button md-icon-button
+        class="mat-paginator-navigation-button mat-paginator-navigation-next"
+        (click)="navigateToNextPage()"
+        [attr.aria-label]="_intl.nextPageLabel"
```

Comment:
Funny thing here, since that works for `md-select` but not `md-button` since `md-select` explicitly marks this as an `@Input` and handles applying the attribute. For the `md-button` however, Angular complains that it is not an input. In the case of aria attributes, they are not properties of the DOM element object and so cannot be applied with just `[aria-label]`